daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -46,14 +47,18 @@ message Unit {}
 message PackageRef {
   oneof Sum {
 
-    // Reference to the package of which the package this referenc
+    // Reference to the package of which the package this reference
     // belongs.
     Unit self = 1;
 
     // A `Package identifier` for an imported Package.
     // A ascii7 lowercase hex-encoded package identifier. This refers
     // to the DAML LF Archive Hash.
     string package_id = 2;
+
+    // An index into `interned_package_ids` of the Package containing
+    // this reference.
+    uint64 interned_id = 3;
```

Comment:
Shouldn't this live in a `oneof` together with `package_id`?

Reply:
It's within one, with `package_id` and `self`.

Reply:
You're totally right. I should have scanned a few more lines. By bad.